### ISSUE TITLE
ZATCA 2nd Phase 

### ISSUE THREAD

**Issue by ismailme33** [mohammadsaif19/zatca_fatoora_flutter#1]
Dear
firstly I want to express my sincere appreciation for your efforts on this work. If it's feasible, could you also extend your support for the second phase of integration.

Thank you dear

**Comment by mohammadsaif19** [comment]
Hala Ismail, 
Thank you for your feedback on the package. However, due to my busy schedule, I can't provide a specific timeline for implementing this feature. If you're interested, feel free to open a pull request.

**Comment by mustafa2011** [comment]
Yes, I'm also very interested to cover phase 2 issue to complete the project as well

**Comment by mohammadsaif19** [comment]
Hala @mustafa2011 ,

I would be more than happy to see your contribution to the implementation of phase 2. Due to my busy schedule, I haven't had enough time to finish it, even though I really wanted to. It's always a pleasure to see contributors like you who are willing to help with open source projects.

**Comment by mustafa2011** [comment]
> Dear firstly I want to express my sincere appreciation for your efforts on this work. If it's feasible, could you also extend your support for the second phase of integration.
> 
> Thank you dear

https://zatca.gov.sa/en/E-Invoicing/Introduction/Guidelines/Documents/E-Invoicing_Detailed__Guideline.pdf

**Comment by mustafa2011** [comment]
> > Dear firstly I want to express my sincere appreciation for your efforts on this work. If it's feasible, could you also extend your support for the second phase of integration.
> > Thank you dear
> 
> https://zatca.gov.sa/en/E-Invoicing/Introduction/Guidelines/Documents/E-Invoicing_Detailed__Guideline.pdf

https://github.com/kpsabithgit/zatcaphase2/tree/main?tab=readme-ov-file
This is also php project covering this issue
